### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by malscent**

### CODE REVIEW

**Review by spjmurray — APPROVED** [review]
links appear to link to something!

File: gcp/couchbase-enterprise-edition-byol/couchbase.py.display [hunk 1]
```diff
@@ -7,10 +7,10 @@ description:
...
-  eulaUrl: 'https://www.couchbase.com/docs/common/terms-of-service.html'
+  eulaUrl: 'https://www.couchbase.com/ESLA08042020'
   documentations:
     - title: Getting Started on the GCP Marketplace
-      url: https://github.com/couchbase-partners/google-deployment-manager-couchbase/README.md
+      url: https://github.com/couchbase-partners/marketplace/blob/main/gcp/shared/README.md
```
Comment (patrick-stephens) [inline gcp/couchbase-enterprise-edition-byol/couchbase.py.display:13]: Should we be versioning this link? What if we change it massively?

**Review by patrick-stephens — APPROVED** [review]
Happy if we don't need to version those links

File: gcp/couchbase-enterprise-edition-byol/couchbase.py.display (hunk 1 above)
Comment (malscent) [reply gcp/couchbase-enterprise-edition-byol/couchbase.py.display:13]: This is the new link to the old document.  It hasn't been changed.  We might want to change it eventually, but again, it's just a link to a "Getting Started" document.  I can always update the link in the future

**Pull request merged**